transformer_engine/pytorch/module/base.py

```diff
@@ -97,10 +98,23 @@ def initialize_ub(
             + "CUDA Multicast. Launch app with UB_SKIPMC=1 to try CUDA IPC instead."
```

Comment:
`assert bool(os.getenv("UB_SKIPMC", "0"))`
This line always pass assertion. "0" will be treated as `True`
it should be `bool(os.getenv("UB_SKIPMC", False))`